print-bootp.c

```diff
@@ -1005,28 +1006,31 @@ rfc1048_print(netdissect_options *ndo,
 				 * uri-length: 2 octets long; specifies the length of the URI data.
 				 * URI: URI of the SZTP bootstrap server.
 				 */
+				uint16_t suboptlen2;
```

Comment:
Placing this declaration inside the `while()` eliminates the need for the additional curly braces.